relay/kusama/src/lib.rs

```diff
@@ -1339,11 +1349,17 @@ impl pallet_message_queue::Config for Runtime {
 	type QueueChangeHandler = ParaInclusion;
 	type QueuePausedQuery = ();
 	type WeightInfo = weights::pallet_message_queue::WeightInfo<Runtime>;
+	type IdleMaxServiceWeight = ();
 }
 
 impl parachains_dmp::Config for Runtime {}
 
 parameter_types! {
+	pub const DefaultChannelSizeAndCapacityWithSystem: (u32, u32) = (
+		// Prepare for 128KiB pages and leave space for a header:
+		1 << 17 - 16,
```

Comment:
@ggwpez the result is `DefaultChannelSizeAndCapacityWithSystem: (2, 10240)`, can you please elaborate more on 128KiB (128*1024=131072) and space for a header? Why `- 16`? Is it a header size in bytes? Don't we have any constant for that? Don't we have also any constant for 128KiB page size?

`1 << 17 - 16 = 2`  (1 x 2^1), should it be `1 << (17 - 16) = 2`  or `(1 << 17) - 16 = 131072 - 16 = 131056`?

The first number in tuple is compared [here](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/runtime/parachains/src/hrmp.rs#L1454-L1464) with `hrmp_channel_max_capacity`, which is **number of messages**:
```
        /// The maximum number of messages allowed in an HRMP channel at once.
	pub hrmp_channel_max_capacity: u32,
```
Btw. actual values for Polkadot are:
```
  hrmpChannelMaxCapacity: 1,000
  hrmpChannelMaxMessageSize: 102,400
```

Also the same for Polkadot bellow where the result is ` `DefaultChannelSizeAndCapacityWithSystem: (1, 1024)`
```
// Prepare for 64KiB pages and leave space for a header:
		1 << 16 - 16,
		1024,
```

---

I just don't understand (probably need more :beers: ):
```
// Prepare for 128KiB pages and leave space for a header:
		1 << 17 - 16,

// Prepare for 64KiB pages and leave space for a header:
		1 << 16 - 16,
```

Reply:
Uhf, good catch. There should be some `()` for sure 😆   
I saw that you added this: https://github.com/paritytech/polkadot-sdk/pull/4332, so i will probably revert here and set it to 50% since the current values are not good. They are slightly larger than any useful page boundary or the Mesage Queue, which we configured to have 64KiB on Polkadot and 128KiB on Kusama.

Reply:
The "preparation" here means that it wont take effect for new channels. So we still need to have some function to reduce the page size of all existing channels.

Reply:
I reverted this for now as the MR is already complicated enough. We can do it in a follow-up.